.github/workflows/pr-ci-build-container-images-cs9.yml

```diff
@@ -0,0 +1,86 @@
+name: "Test build of CentOS Stream 9 based container images"
+on:
+  pull_request:
+    branches:
+      - master
+jobs:
+  test-build-image-operator:
+    runs-on: ubuntu-20.04
+    env:
+      IMAGE_NAME: ghcr.io/${{ github.repository }}
+    steps:
+      - name: Check out the repo
+        uses: actions/checkout@v2
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v1
+
+      - name: Login to Docker
+        uses: docker/login-action@v1
+        with:
+          registry: ghcr.io
+          username: ${{ github.repository_owner }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Build and push sriov-network-operator
+        uses: docker/build-push-action@v2
+        with:
+          context: .
+          push: false
+          tags: |
+            ${{ env.IMAGE_NAME }}:latest
+          file: ./Dockerfile
```

Comment:
Shall we use the `.el9` dockerfiles?
Or does it need to wait until this PR merged, otherwise CI would fail for this PR?